controllers/cards.js

```diff
@@ -1,26 +1,35 @@
 const Card = require('../models/card.js');
+const BadRequestError = require('../errors/badRequestError');
+const NotFoundError = require('../errors/notFoundError');
+const ForbiddenError = require('../errors/forbiddenError');
 
 module.exports.getCards = (req, res) => {
   Card.find({})
     .then((card) => res.send({ data: card }))
     .catch((err) => res.status(500).send({ message: err.message }));
 };
 
-module.exports.createCard = (req, res) => {
+module.exports.createCard = (req, res, next) => {
   const { name, link } = req.body;
   const owner = req.user._id;
   Card.create({ name, link, owner })
     .then((card) => res.send({ data: card }))
-    .catch((err) => res.status(500).send({ message: err.message }));
+    .catch(() => next(new BadRequestError('Данные не прошли валидацию')));
```

Comment:
Нужно исправить: при обработке ошибок ошибка в консоли.
Ошибка:
```bash
Error: Данные не прошли валидацию
    at Card.create.then.catch (/Users/jmi/work/ya-reviews/mestoBackend/controllers/cards.js:17:23)
    at process._tickCallback (internal/process/next_tick.js:68:7)
```

Валидация данных пройдет, при сохранении записи модель запускает валидацию согласно схеме. Но чтобы был отправлен именно 400 код, нужно делать немного по-другому, есть много способов, но один из лучших - валидировать данные во время роутинга (те до передачи их в контроллер). Как это делать будет рассказано в следующей практике.

Здесь же нужно оставить как и было:
```suggestion
    .catch((err) => res.status(500).send({ message: err.message }));
```

Также нужно обязательно не забыть удалить объявление `next`.